### PR TITLE
Initial support for Windows

### PR DESCRIPTION
**Pull request by Omico**

### CODE REVIEW

**Comment by Omico** [comment]
Just notice we can use `org.jetbrains.jewel.window.utils.DesktopPlatform` to check what platform users use. Do I need to migrate to that?

**Comment by Omico** [comment]
Seems `org.jetbrains.jewel.window.TitleBar` doesn't work correctly on Windows. 

image

If we remove it, everything looks fine.

image

File: src/jvmMain/kotlin/dev/romainguy/kotlin/explorer/OS.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package dev.romainguy.kotlin.explorer
```
Comment (romainguy) [inline src/jvmMain/kotlin/dev/romainguy/kotlin/explorer/OS.kt:1]: Add missing copyright header please, otherwise looks good, thanks!

**Review by romainguy — CHANGES_REQUESTED** [review]

**Comment by Omico** [comment]
Do you want me to add Spotless to do that automatically in another PR?

**Comment by romainguy** [comment]
Thanks for the suggestion but I'm not fond of auto formatters :)

Thanks for the PR. I'll try to keep Windows in mind but I won't be able to regularly test future changes :/

**Pull request merged**